src/LocalLogins.php

```diff
@@ -20,7 +19,7 @@ public function register(Panel $panel): void
         }
 
         $panel
-            ->login(LoginPage::class)
+            ->login(config("filament-local-logins.panels.{$panel->getId()}.login_page"))
```

Comment:
We need to have a default option here so that this doesn't become a breaking change for users.